From 4bdeeb740499e0454627f6d97531fa8c85f213ec Mon Sep 17 00:00:00 2001
From: Mateusz Stadnik <matgla@live.com>
Date: Fri, 5 Sep 2025 22:10:59 +0200
Subject: [PATCH] fixed incorrect array size

---
 toys/pending/sh.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/toys/pending/sh.c b/toys/pending/sh.c
index 63026b99..47c4c1a2 100644
--- a/toys/pending/sh.c
+++ b/toys/pending/sh.c
@@ -3040,7 +3040,8 @@ if (DEBUG) { dprintf(2, "%d run command %p %s\n", getpid(), TT.ff, *pp->arg.v);
 // TODO: when to prioritize $PATH over MAYFORK?
     if (jj&(TOYFLAG_NOFORK|TOYFLAG_MAYFORK)) {
       sigjmp_buf rebound, *prebound = toys.rebound;
-      char temp[jj = offsetof(struct toy_context, rebound)];
+      jj = offsetof(struct toy_context, rebound);
+      char temp[256];
 
       // This fakes lots of what toybox_main() does.
       memcpy(&temp, &toys, jj);
-- 
2.39.5

